client/components/chart/index.jsx

```diff
@@ -60,29 +61,27 @@ module.exports = React.createClass( {
 	},
 
 	resize: function() {
-		if ( this.isMounted() ) {
```

Comment:
To fix block scope issues, I'd recommend turning this into an early return instead:
```javascript
if ( ! this.isMounted() ) {
    return;
}
```